# connectiontcpserver.cpp: error: 'memset' was not declared in this scope

--- src/connectiontcpserver.cpp.orig	2019-12-09 03:14:10.000000000 +0800
+++ src/connectiontcpserver.cpp	2022-10-27 05:43:11.000000000 +0800
@@ -51,7 +51,7 @@
 #endif
 
 #include <cstdlib>
-#include <string>
+#include <cstring>
 
 #ifndef _WIN32_WCE
 # include <sys/types.h>
